tosca-templates/htcondor_cluster.yml

```diff
@@ -78,7 +78,7 @@ topology_template:
       type: tosca.nodes.indigo.LRMS.FrontEnd.HTCondor
       properties:
         wn_ips: { get_attribute: [ lrms_wn, private_address ] }
-        htcondor_password: { get_input: htcondor_password }C
+        htcondor_password: { get_input: htcondor_password }
```

Comment:
Interesting.

I am testing this in IM-dev, and I get:

```bash
TASK [grycap.htcondor : Store Condor credentials] ******************************
Wednesday 06 September 2023  11:56:32.961011 
fatal: [192.168.13.48_3]: FAILED! =&gt; {"msg": "An unhandled exception occurred while templating '{{ htcondor_password }}'. Error was a <class 'ansible.errors.ansibleerror'="">, original message: An unhandled exception occurred while templating '{{ htcondor_password }}'.
```
However, the ansible role works well when I use it out of IM.

Do you know why?

Reply:
oh wait, should I wait for `grycap.htcondor` to be updated in https://galaxy.ansible.com/grycap/htcondor, right?

Reply:
Interesting... 

How come that you don't get the ` "An unhandled exception occurred while templating '{{ htcondor_password }}'` error?

Reply:
I did another test. I think the `htcondor_password` issue comes when you change the default password in the IM-dev web interface, under the `HTCondor configuration` tab, before clicking `Submit`?

Reply:
> I did another test. I think the `htcondor_password` issue comes when you change the default password in the IM-dev web interface, under the `HTCondor configuration` tab, before clicking `Submit`?

May be, as in my test I did not change the default value.
Let me check.

Reply:
I tested changing the password and it works.
Maybe you are using an invalid character.